dev/packages/beats/mysql-0.0.1/dataset/status/manifest.yml

```diff
@@ -8,7 +8,13 @@ streams:
   - default:
     - root:secret@tcp(127.0.0.1:3306)/
     name: hosts
+  - default: secret
```

Comment:
I think we are missing here all the additional fields which are need to make the UI nicer. See https://github.com/elastic/package-registry/blob/master/dev/packages/example/nginx-1.2.0/dataset/access/manifest.yml#L17 for an example.

Reply:
So this is mostly manual work here. Once I fix it, it would be harder to automatically import beats. Should all options have these properties?

Reply:
https://github.com/elastic/package-registry/issues/318

Reply:
I think most fields will need it, at least part of the options. And yes, this will probably rely on manual work as it is not something we have today.